src/main/java/cms/gov/madie/terminology/controller/VsacController.java

```diff
@@ -0,0 +1,55 @@
+package cms.gov.madie.terminology.controller;
+
+import org.springframework.web.bind.annotation.PostMapping;
+
+import java.util.concurrent.ExecutionException;
+
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import cms.gov.madie.terminology.service.VsacService;
+
+import lombok.extern.slf4j.Slf4j;
+import generated.vsac.nlm.nih.gov.RetrieveMultipleValueSetsResponse;
+
+@RestController
+@RequestMapping(path = "/vsac")
+@Slf4j
+public class VsacController {
+
+  private final VsacService vsacService;
+
+  public VsacController(VsacService vsacService) {
+    this.vsacService = vsacService;
+  }
+
+  @PostMapping(path = "/serviceTicket", produces = "text/plain")
+  public ResponseEntity<String> getServiceTicket(
+      @RequestParam(required = true, name = "tgt") String tgt)
+      throws InterruptedException, ExecutionException {
+    log.info("Entering: getServiceTicket(): tgt = " + tgt);
+    String st = vsacService.getServiceTicket(tgt);
+    return ResponseEntity.ok().body(st);
+  }
+
+  @GetMapping(
+      path = "/valueSet",
+      consumes = "text/plain",
+      produces = MediaType.APPLICATION_XML_VALUE)
+  public ResponseEntity<RetrieveMultipleValueSetsResponse> getValueSet(
+      @RequestParam(required = true, name = "oid") String oid,
+      @RequestParam(required = true, name = "stNumber") String stNumber,
```

Comment:
How about using the client to obtain the ST instead of having the consumer call the service twice?

Could either overload the endpoint to accept the TGT instead of the ST, or replace the ST param with the TGT.